### PR TITLE
Added python support for wgs84 transforms

### PR DESCRIPTION
**Pull request by agyoungs**
This gives basic functionality to convert local coordinates to WGS84 coordinates and vice versa in python.

### CODE REVIEW

File: swri_transform_util/src/swri_transform_util/wgs84_transformer.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+import math
+import numpy as np
```
Comment (pjreed) [inline swri_transform_util/src/swri_transform_util/wgs84_transformer.py:34]: It doesn't look like there's currently a dependency for numpy in the package.xml, so you'll want to add an `exec_depend` for that.  Also make sure to check the Python version and depend on `python-numpy` for Python 2 and `python3-numpy` for Python 3.  The package.xml should have an example of doing that for setuptools.

File: swri_transform_util/src/swri_transform_util/wgs84_transformer.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+    def __init__(self, local_origin):
+        """
+        Constructor for the Wgs84Transformer class
+        :param geometry_msgs.Pose local_origin: An initialized local origin
+        """
+        self._reference_heading = -1.0 * euler_from_quaternion(quaternion=(local_origin.pose.orientation.x,
+                                                                           local_origin.pose.orientation.y,
+                                                                           local_origin.pose.orientation.z,
+                                                                           local_origin.pose.orientation.w))[2]  # get yaw from quaternion
```
Comment (pjreed) [inline swri_transform_util/src/swri_transform_util/wgs84_transformer.py:58]: PEP-8 requires lines be < 100 characters long.
```suggestion
        self._reference_heading = -1.0 * euler_from_quaternion(
            quaternion=(local_origin.pose.orientation.x,
                        local_origin.pose.orientation.y,
                        local_origin.pose.orientation.z,
                        local_origin.pose.orientation.w))[2]  # get yaw from quaternion
```

File: swri_transform_util/src/swri_transform_util/wgs84_transformer.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        """
+        Transforms point(s) in the WGS84 coordinate frame to the local_xy frame.
+        :param list wgs84_points: pair-wise list of (latitude, longitude) coordinates
+        :return: A pair-wise list of (x, y) coordinates in the local_xy frame
+        """
+        wgs84_points = np.array(wgs84_points)
+
+        r = wgs84_points * math.pi / 180.0
+
+        d = (r - [self._reference_latitude, self._reference_longitude]) * [self._rho_lat, self._rho_lon]
```
Comment (pjreed) [inline swri_transform_util/src/swri_transform_util/wgs84_transformer.py:86]:
```suggestion
        d = (r - [self._reference_latitude, self._reference_longitude]) * [self._rho_lat,
                                                                           self._rho_lon]
```

File: swri_transform_util/src/swri_transform_util/wgs84_transformer.py [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+        points = d.dot([[-self._sin_heading, self._cos_heading], [self._cos_heading, self._sin_heading]])
```
Comment (pjreed) [inline swri_transform_util/src/swri_transform_util/wgs84_transformer.py:88]:
```suggestion
        points = d.dot([[-self._sin_heading, self._cos_heading],
                        [self._cos_heading, self._sin_heading]])
```

File: swri_transform_util/src/swri_transform_util/wgs84_transformer.py [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+        """
+        Transforms point(s) in the local_xy frame to the WGS84 coordinate frame
+        :param list local_points: pair-wise list of (x, y) coordinates
+        :return: A pair-wise list of (latitude, longitude) coordinates in the WGS84 frame
+        """
+        points = np.array(local_points)
+
+        d = points.dot([[self._sin_heading * self._cos_heading, self._cos_heading],
+                        [self._sin_heading**2, self._sin_heading]])
+        d[:,0] = (points[:,1] - d[:,0]) / self._cos_heading
```
Comment (pjreed) [inline swri_transform_util/src/swri_transform_util/wgs84_transformer.py:102]: PEP 8 also requires spaces after commas.
```suggestion
        d[:, 0] = (points[:, 1] - d[:, 0]) / self._cos_heading
```

File: swri_transform_util/src/swri_transform_util/wgs84_transformer.py [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+        r = d / [self._rho_lat, self._rho_lon] + [self._reference_latitude, self._reference_longitude]
```
Comment (pjreed) [inline swri_transform_util/src/swri_transform_util/wgs84_transformer.py:104]:
```suggestion
        r = d / [self._rho_lat, self._rho_lon] + [self._reference_latitude,
                                                  self._reference_longitude]
```

**Comment by agyoungs** [comment]
> Would it also be possible to cherry-pick this into the `dashing-devel` branch and also open up a PR for ROS2 support?

#596 has the cherry-picked changes

**Pull request merged**